specs.md

```diff
@@ -7,6 +7,6 @@ layout: default
 - Embedding interfaces:
   - [JavaScript API](https://webassembly.github.io/spec/js-api/index.html): defines JavaScript classes and objects for accessing WebAssembly from within JavaScript, including methods for validation, compilation, instantiation, and classes for representing and manipulating imports and exports as JavaScript objects.
   - [Web API](https://webassembly.github.io/spec/web-api/index.html): defines extensions to the JavaScript API made available specifically in web browsers, in particular, an interface for streaming compilation and instantiation from origin-bound `Response` types.
-  - [WASI API](https://github.com/WebAssembly/WASI/blob/master/phases/snapshot/docs.md): defines a modular system interface to run WebAssembly outside the web, providing access to things like files, network connections, clocks, and random numbers.
+  - [WASI API](https://github.com/WebAssembly/WASI/blob/main/Proposals.md): defines a modular system interface to run WebAssembly outside the web, providing access to things like files, network connections, clocks, and random numbers.
```

Comment:
Lets link to the https://github.com/WebAssembly/WASI markdown file. It has a link to the proposals as well, but also includes a detailed description